test_exercise30.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
```

Comment:
@paulghaddad This test passes as it's written. However, do I need to hard-code the long multiplication table into this test (ie, 
0 *0 = 0
0 * 1 = 0
0 *2 = 0
...)

If so, I'm having a tough time getting this long list to pass the test. Originally, I put the result as the second argument when I call self.assert:
self.assertEqual(complete_multiplication_table, 0 *0 = 0
                                                                                 0 * 1 = 0
                                                                                 ...)
But I get an error here. I also tried putting this table as a variable and passing in the variable, but that also resulted in an error. Thoughts on this?